src/util/minmax_heap.c

```diff
@@ -14,7 +14,9 @@
 
 #include "rmalloc.h"
 
-#define is_min(n) ((log2_32(n) & 1) == 0)
+// parity of the number of leading zeros, AKA parity of log2(n)
```

Comment:
Good. But for future proofing, please add another couple words explaining why this is correct.